src/main/java/github/scarsz/discordsrv/util/DebugUtil.java

```diff
@@ -207,11 +207,10 @@ private static String getRelevantLinesFromServerLog() {
             while (!done) {
                 String line = br.readLine();
                 if (line == null) done = true;
-                if (line != null
-                        && line.toLowerCase().contains("discordsrv")
-                        && !line.toLowerCase().contains("[discordsrv] chat:")) {
-                    output.add(DiscordUtil.aggressiveStrip(line));
-                }
+                else if (
+                    (line.toLowerCase().contains("discordsrv") && !line.toLowerCase().contains("[discordsrv] chat:"))
+                    || line.toLowerCase().contains(" /discord")
```

Comment:
Is the extra space here intentional?

Reply:
Yes, otherwise some URLs, class paths, and potentionally more would be unintentionally matched.

Reply:
Gotcha, I was making sure bc of URLs, didn't think about class paths though